internal/examples/server/opampsrv/opampsrv.go

```diff
@@ -54,6 +56,7 @@ func (srv *Server) Start() {
 			},
 		},
 		ListenEndpoint: "127.0.0.1:4320",
+		HTTPMiddleware: otelhttp.NewMiddleware("HTTP opamp-go"),
```

Comment:
Nit: [http semconv](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md) says span name should be `{method} {http.route}`. Our default route is `/v1/opamp`. Method is tricky, I think it is "GET" for websocket and "POST" for plain http and we don't here which it will be, so perhaps we skip it and use just the route, e.g.
```suggestion
		HTTPMiddleware: otelhttp.NewMiddleware("/v1/opamp"),
```

Reply:
Good call! I wasn't sure what to put here and keep forgetting to check semcov's